tests/simplex/cell_measure_01.output

```diff
@@ -1,21 +1,53 @@
 
 DEAL::dim=2 spacedim=2:
-DEAL::measure:  0.500000
-DEAL::diameter: 1.41421
-DEAL::barycenter: 0.333333 0.333333
-DEAL::diameter_min: 1.41421
-DEAL::diameter_max: 1.41421
-DEAL::
-DEAL::dim=3 spacedim=3:
-DEAL::measure:  0.166667
-DEAL::diameter: 1.41421
-DEAL::barycenter: 0.250000 0.250000 0.250000
-DEAL::diameter_min: 1.41421
-DEAL::diameter_max: 1.41421
-DEAL::
-DEAL::dim=3 spacedim=3:
-DEAL::measure:  0.500000
-DEAL::diameter: 1.73205
-DEAL::diameter_min: 1.73205
-DEAL::diameter_max: 1.73205
+DEAL::measure: 0.500000
+DEAL::measure (via quadrature): 0.500000
+DEAL::diameter 1.41421
+DEAL::barycenter 0.333333 0.333333
+DEAL::diameter_min 1.41421
+DEAL::diameter_max 1.41421
 DEAL::
+DEAL:basic tetrahedron::dim=3 spacedim=3:
+DEAL:basic tetrahedron::measure: 0.166667
+DEAL:basic tetrahedron::measure (via quadrature): 0.166667
```

Comment:
I didn't verify these results by hand - I am assuming that since we get the same values with the measure formula and quadrature that this measure formula is correct.

Reply:
Well, at least, for the arbitrary objects below - for the unit shapes its clear that they give the correct results.